src/coverlet.msbuild.tasks/InstrumentationTask.cs

```diff
@@ -17,6 +17,7 @@ public class InstrumentationTask : Task
         private bool _singleHit;
         private string _mergeWith;
         private bool _useSourceLink;
+        private readonly MSBuildLogger _logger;
```

Comment:
remove `readonly` for now, maybe on some next PR we should change for in all codebase